source/agora/test/EnrollmentManager.d

```diff
@@ -26,71 +26,7 @@ import agora.consensus.EnrollmentManager;
 import agora.consensus.Genesis;
 import agora.test.Base;
 
-/// test enrollment process
```

Comment:
Why was this removed ?

Reply:
The unit test code has been merged into the code below the comment `/// test for  enrollment process & revealing a pre-image periodically`. Changing the way the EnrollmentManager stores the data make the test code need to be changed and I found there is redundant test code, I removed the redundant code and merged the part of it into the other test code.